### PR TITLE
Move properties into bottom drawer

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
Properties:

- Add new properties file
- Delete nested properties

Layout Adjustments:

- IFC properties right drawer on desktop
- IFC properties bottom drawer on mobile
- Icon button group

Icons: 

- SVGs for on/off states

https://olegmoshkovich.github.io/Share/

### CODE REVIEW

File: src/Components/ItemProperties.jsx [hunk 1]
```diff
@@ -17,28 +17,34 @@ const row = (d1, d2, serial) => {
...
         {d1}
       </td>
       <td
         style={{
+          maxWidth:'200px',
+          overflowWrap: 'break-word',
           border: '1px solid lightgrey',
           fontFamily: 'Helvetica',
-          fontSize: 14,
+          fontSize: 12,
```
Comment (pablo-mayrgundter) [inline src/Components/ItemProperties.jsx:43]: '12px' like above

File: src/Components/ItemProperties.jsx [hunk 2]
```diff
@@ -146,19 +132,23 @@ const typeValue = (obj) => {
 };
 
 const ObjectTree = ({ name, obj }) => {
+  console.log('object', obj)
+  console.log('in the object tree')
```
Comment (pablo-mayrgundter) [inline src/Components/ItemProperties.jsx:136]: remove debugging

File: src/Components/ItemPropertiesV1.jsx [hunk 3]
```diff
@@ -0,0 +1,165 @@
+import TreeItem from '@mui/lab/TreeItem';
```
Comment (pablo-mayrgundter) [inline src/Components/ItemPropertiesV1.jsx:1]: I'm generally not a fan of versioned filenames.  Why need this?

File: src/Components/SideMenu.jsx [hunk 4]
```diff
@@ -2,117 +2,84 @@ import React from 'react';
...
-    marginRight: 30,
+  drawerPaper:{
+    marginTop:'65px',
+    height:'200px',
+    width:'350px',
+    borderRadius:'0px',
+    marginLeft:'20px',
+    zIndex: 10,
+    overflow:'hidden',
+    '@media (max-width: 900px)': {
```
Comment (pablo-mayrgundter) [inline src/Components/SideMenu.jsx:15]: Pls indent new curly scope region.  And below

File: src/Containers/CadView.jsx [hunk 5]
```diff
@@ -373,7 +378,10 @@ const CadView = () => {
               viewer={viewer}
               element={selectedElement}
               close = {()=>setShowItemPanel(false)}
-              topOffset = {PANEL_TOP}/>
+              topOffset = {PANEL_TOP}
+              placeCutPlane = {()=>placeCutPlane()}
```
Comment (pablo-mayrgundter) [inline src/Containers/CadView.jsx:382]: Not necessary here, but we need to think of a quick way to factor these out of CadView, for improved latency.

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]
Looks good overall.. a few comments to fix.

File: src/Components/ItemPropertiesV1.jsx (hunk 3 above)
Comment (OlegMoshkovich) [reply src/Components/ItemPropertiesV1.jsx:1]: removed

**Review by OlegMoshkovich — COMMENTED** [review]
Can you please approve?

**Pull request merged**